eos-tech-support/eos-enable-coredumps

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# Enables coredumps and adjusts storage path to work with coredumpcl
+
+if [ $# -lt 1 ] || [ $1 == "-h" ] || [ $1 == "--help" ]; then
+    echo "Usage:"
+    echo "   `basename $0` sysconfdir"
+    echo "Where:"
+    echo "   sysconfdir = path to the system configuration (e.g. /etc)"
+    exit
+fi
+
+SYSCONFDIR=${1}
+
+CONF_FILENAME=90-coredump.conf
```

Comment:
The pedant in me is saying that instead of hard-coding `90` we should have this script work out the first unused number, and use that, but I think that would over-complicate things. (Don’t make it do that unless you’re a fanatic. If you do make it do that, rewrite in Python because you’re a fanatic, and because doing that in bash would be horrific.)

Reply:
I'm not a fanatic, and not swimming in time at the moment to rewrite this really :-)

Besides, the number was hardcoded already and using things like `50-`, `90-` is common practice anyway so I think it's good enough as an improvement over what we had. Beside, I'm not convinced you really want to work out that number automatically either in all cases.